packages/clarity-js/src/layout/node.ts

```diff
@@ -157,6 +158,21 @@ export default function (node: Node, source: Source): Node {
                     }
                     dom[call](node, parent, frameData, source);
                     break;
+                case "LINK":
+                    // electron stylesheets reference the local file system - translating those
+                    // to inline styles so playback can work
+                    if (electron && attributes['rel'] === 'stylesheet') {
```

Comment:
>stylesheet [](http://example.com/codeflow?start=59&length=10)

Worth moving this to Constant in d.ts file and then use it here like some of the other constants in this file.